python/jsbeautifier/cli/__init__.py

```diff
@@ -158,9 +146,7 @@ def get_filepaths_from_params(filepath_params, replace):
         elif "*" in filepath_param or "?" in filepath_param:
             # handle globs
             # empty result is okay
-            if sys.version_info.major == 2 or (
-                sys.version_info.major == 3 and sys.version_info.minor <= 4
-            ):
+            if sys.version_info.major == 3 and sys.version_info.minor <= 4:
```

Comment:
```suggestion
            if sys.version_info <= (3, 4):
```
https://docs.astral.sh/ruff/rules/#flake8-2020-ytt

Reply:
I know. I don't know the baseline of this project.